mithril-aggregator/Cargo.toml

```diff
@@ -43,13 +44,7 @@ paste = "1.0.15"
 percent-encoding = "2.3.2"
 rayon = { workspace = true }
 regex = "1.12.2"
-reqwest = { workspace = true, features = [
-    "default",
-    "gzip",
-    "zstd",
-    "deflate",
-    "brotli"
-] }
+reqwest = { workspace = true, features = ["default"] }
```

Comment:
Maybe we still need the traffic compression features which are used e.g. to retrieve the mapping of pool ids to the pool names by cexplorer.